Source/WebKit/UIProcess/ios/PageClientImplIOS.mm

```diff
@@ -76,7 +76,7 @@
 #import <wtf/BlockPtr.h>
 #import <wtf/cocoa/Entitlements.h>
 
-#define MESSAGE_CHECK(assertion) MESSAGE_CHECK_BASE(assertion, m_webView.get()->_page->process().connection())
+#define MESSAGE_CHECK(assertion) MESSAGE_CHECK_BASE(assertion, webView()->_page->process().connection())
```

Comment:
Wasn't sure the best way to add a `nil` check here, or if it was even needed (if `m_webView` should never be `nil` when this is used).